src/style.css

```
@@ -0,0 +1,68 @@
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.snake {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.box {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+  flex-basis: 30%;
+  min-width: 300px;
+  height: 300px;
+  color: white;
+  font-size: 100px;
+}
+
+.box-1 {
+  background-color: #f00;
+}
+
+.box-2 {
+  background-color: #c00;
+}
+
+.box-3 {
+  background-color: #900;
+}
+
+.box-4 {
+  background-color: #600;
+}
+
+.box-5 {
+  background-color: #300;
+}
+
+.box-6 {
+  background-color: #000;
+}
+
+@media (min-width: 900px) {
+  .box-4 {
+    order: 2;
+  }
+}
+
+@media (min-width: 900px) {
+  .box-5 {
+    order: 1;
+  }
+}
+
+@media (min-width: 600px) and (max-width: 890px) {
+  .box-3 {
+    order: 1;
+  }
+
+  .box-5,
+  .box-6 {
+    order: 2;
+  }
+}
```

maybe use min-width: 890px in the previous media instead of 900px

```suggestion
@media (min-width: 600px) {
  .box-3 {
    order: 1;
  }

  .box-5,
  .box-6 {
    order: 2;
  }
}
```